src/App.js

```diff
@@ -1,24 +1,70 @@
 import React from 'react';
 import './App.scss';
 
-const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+class App extends React.Component {
+  state = {
+    goodsFromServer: [
```

Comment:
```suggestion
  state = {
    goods: [...goodsFromServer]
```
No need to write the whole array in the state. You could create a copy for this